src/main/java/stsjorbsmod/cards/wanderer/Firebolt.java

```diff
@@ -25,31 +30,39 @@
 
     private static final int COST = 1;
     private static final int DAMAGE = 8;
-    private static final int DAMAGE_PER_CLARITY = 2;
-    private static final int UPGRADE_PLUS_DAMAGE_PER_CLARITY = 1;
+    private static final int AMOUNT_PER_CLARITY = 2;
 
     public Firebolt() {
         super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
-        damage = baseDamage = DAMAGE;
-        magicNumber = baseMagicNumber = DAMAGE_PER_CLARITY;
+        damage = baseDamage = metaMagicNumber = baseMetaMagicNumber = DAMAGE;
+        magicNumber = baseMagicNumber = AMOUNT_PER_CLARITY;
     }
 
     @Override
     protected int calculateBonusBaseDamage() {

```

Comment:
I think you can just omit `calculateBonusBaseDamage` entirely with this design update

Reply:
oh, misread which card was which, oops